src/main/scala/calc/stdlib/Math.scala

```diff
@@ -0,0 +1,27 @@
+package calc.stdlib
+
+import calc.{FunctionType, TreeType}
+
+/** Some simple mathematical functions to use with the language. */
+object Math {
+
+  val moduleName: String = "jl_Math$"
+
+  val functionTypes: Map[String, TreeType] = Map(
+    "abs"   -> FunctionType(1),
+    "acos"  -> FunctionType(1),
+    "asin"  -> FunctionType(1),
+    "atan"  -> FunctionType(1),
+    "ceil"  -> FunctionType(1),
+    "cos"   -> FunctionType(1),
+    "cosh"  -> FunctionType(1),
+    "exp"   -> FunctionType(1),
+    "floor" -> FunctionType(1),
+    "log"   -> FunctionType(1),
+    "round" -> FunctionType(1),
+    "sin"   -> FunctionType(1),
+    "sinh"  -> FunctionType(1),
+    "tan"   -> FunctionType(1),
+    "tanh"  -> FunctionType(1)
```

Comment:
It seems your implementation is generic wrt the arity of math functions, but you only define math functions of arity 1. Consider adding `pow` as an example of a math function with more than 1 argument.